moveit_planners/chomp/chomp_optimizer_adapter/chomp_optimizer_adapter_plugin_description.xml

```diff
@@ -0,0 +1,9 @@
+<library path="libmoveit_chomp_optimizer_adapter">
+
+  <class name="chomp/OptimizerAdapter" type="chomp::OptimizerAdapter"
```

Comment:
:+1: for the name change. It's simply not a "default".

[The tutorial](https://ros-planning.github.io/moveit_tutorials/doc/chomp_planner/chomp_planner_tutorial.html) has to be changed to the new name though.

Reply:
This is on my (long) list...